src/rust/ide/view/graph-editor/src/component/visualization/foreign/java_script/instance.rs

```diff
@@ -231,6 +229,15 @@ impl Instance {
                     frp.data_receive_error.emit(Some(e));
                 }
              });
+
+            mouse_up       <- scene.mouse.frp.up.constant(());
+            mouse_down     <- scene.mouse.frp.down.constant(());
+            mouse_wheel    <- scene.mouse.frp.wheel.constant(());
+            mouse_position <- scene.mouse.frp.position.constant(());
+            caught_mouse   <- any(mouse_up,mouse_down,mouse_wheel,mouse_position);
```

Comment:
I believe you dont need these constants and you can use `any_` instead which just drops all input data types and emits empty tuple instead

Reply:
Will `any_` work with different types?

Reply:
The answer is: yes, it will.